geomesa-arrow/geomesa-arrow-gt/src/main/scala/org/locationtech/geomesa/arrow/vector/ArrowAttributeWriter.scala

```diff
@@ -464,13 +465,21 @@ object ArrowAttributeWriter {
   /**
     * Writes geometries - delegates to our JTS geometry vectors
     */
-  class ArrowGeometryWriter(val name: String, delegate: GeometryVector[Geometry, FieldVector])
+  class ArrowGeometryWriter(val name: String, delegate: GeometryVector[Geometry, FieldVector], encoding: SimpleFeatureEncoding)
       extends ArrowAttributeWriter {
 
     override def vector: FieldVector = delegate.getVector
 
     // note: delegate handles nulls
-    override def apply(i: Int, value: AnyRef): Unit = delegate.set(i, value.asInstanceOf[Geometry])
+    override def apply(i: Int, value: AnyRef): Unit = {
+      val options = Map("axisOrder" -> (encoding.axisOrder match {
+        case Encoding.Max => AxisOrder.EAST_NORTH /* Lon/Lat */
+        case _ => AxisOrder.NORTH_EAST /* Lat/Lon */
+      })).asInstanceOf[Map[String, AnyRef]].asJava
+
+      delegate.setOptions(options)
```

Comment:
instead of an untyped map, could we just have a `setLongitudeFirst(bool)` method? that way, we get compile time safety, and don't need to worry about magic keys and type casting, etc.
this should also be called just once, probably up on line 272 when the writer is created.

Reply:
Was originally going for maximum flexibility and to provide a mechanism to snake in any additional options / metadata to the geometry encoders.